{"name": "test_func2", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_func2():\n>       assert False\nE       assert False\n\ntest_wallet.py:11: AssertionError"}, "attachments": [{"name": "stdout", "source": "0fa9fefc-631c-4223-a381-78310d855f10-attachment.txt", "type": "text/plain"}], "start": 1678268173120, "stop": 1678268173121, "uuid": "d9a28187-407c-43cf-b4a8-baac84843f6f", "historyId": "24e6e07915fb23c6ba8aab2e357abe9b", "testCaseId": "3e882ca875abd23429202ae8e442cea9", "fullName": "test_wallet#test_func2", "labels": [{"name": "suite", "value": "test_wallet"}, {"name": "host", "value": "BAN-LAP-VREDDY"}, {"name": "thread", "value": "7936-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_wallet"}]}